countries_regions_phase1_fix.csv

```diff
@@ -0,0 +1,237 @@
+Burundi,
+Cambodia,
+Cameroon,
+Canada,
+Cape Verde,
+Central African Republic,
+Chad,
+Chile,
+China,
+Colombia,
+Comoros,
+Congo,
+Costa Rica,
+Cote d'Ivoire,
+Croatia,
+Cuba,
+Cyprus,
+Czech Republic,
+Democratic Republic of Congo,
+Denmark,
+Djibouti,
+Dominica,
+Dominican Republic,
+Ecuador,
+Egypt,
+El Salvador,
+Eritrea,
+Estonia,
+Eswatini,
+Ethiopia,
+Faeroe Islands,
+Fiji,
+Finland,
+France,
+Gabon,
+Gambia,
+Georgia,
+Germany,
+Ghana,
+Greece,
+Greenland,
+Guam,
+Guatemala,
+Guinea,
+Guyana,
+Haiti,
+Honduras,
+Hong Kong,
+Hungary,
+Iceland,
+India,
+Indonesia,
+Iran,
+Iraq,
+Ireland,
+Israel,
+Italy,
+Jamaica,
+Japan,
+Jordan,
+Kazakhstan,
+Kenya,
+Kosovo,
+Kuwait,
+Kyrgyz Republic,
+Laos,
+Latvia,
+Lebanon,
+Lesotho,
+Liberia,
+Libya,
+Lithuania,
+Luxembourg,
+Macao,
+Madagascar,
+Malawi,
+Malaysia,
+Mali,
+Mauritania,
+Mauritius,
+Mexico,
+Moldova,
+Monaco,
+Mongolia,
+Morocco,
+Mozambique,
+Myanmar,
+Namibia,
+Nepal,
+Netherlands,
+New Zealand,
+Nicaragua,
+Niger,
+Nigeria,
+Norway,
+Oman,
+Pakistan,
+Palestine,
+Panama,
+Papua New Guinea,
+Paraguay,
+Peru,
+Philippines,
+Poland,
+Portugal,
+Puerto Rico,
+Qatar,
+Romania,
+Russia,
+Rwanda,
+San Marino,
+Saudi Arabia,
+Senegal,
+Serbia,
+Seychelles,
+Sierra Leone,
+Singapore,
+Slovak Republic,
+Slovenia,
+Solomon Islands,
+Somalia,
+South Africa,
+South Korea,
+South Sudan,
+Spain,
+Sri Lanka,
+Sudan,
+Suriname,
+Sweden,
+Switzerland,
+Syria,
+Taiwan,
+Tajikistan,
+Tanzania,
+Thailand,
+Timor-Leste,
+Togo,
+Trinidad and Tobago,
+Tunisia,
+Turkey,
+Uganda,
+Ukraine,
+United Arab Emirates,
+United Kingdom,
+United Kingdom,England
+United Kingdom,Northern Ireland
+United Kingdom,Scotland
+United Kingdom,Wales
+United States,
+United States,Alabama
+United States,Alaska
+United States,Arizona
+United States,Arkansas
+United States,California
+United States,Colorado
+United States,Connecticut
+United States,Delaware
+United States,Florida
+United States,Georgia
+United States,Hawaii
+United States,Idaho
+United States,Illinois
+United States,Indiana
+United States,Iowa
+United States,Kansas
+United States,Kentucky
+United States,Louisiana
+United States,Maine
+United States,Maryland
+United States,Massachusetts
+United States,Michigan
+United States,Minnesota
+United States,Mississippi
+United States,Missouri
+United States,Montana
+United States,Nebraska
+United States,Nevada
+United States,New Hampshire
+United States,New Jersey
+United States,New Mexico
+United States,New York
+United States,North Carolina
+United States,North Dakota
+United States,Ohio
+United States,Oklahoma
+United States,Oregon
+United States,Pennsylvania
+United States,Rhode Island
+United States,South Carolina
+United States,South Dakota
+United States,Tennessee
+United States,Texas
+United States,Utah
+United States,Vermont
+United States,Virginia
+United States,Washington
+United States,Washington DC
+United States,West Virginia
+United States,Wisconsin
+United States,Wyoming
+United States Virgin Islands,
```

Comment:
This is the list of countries from latest Oxford dataset we should consider for Phase 1. The new name of US Virgin Islands is `United States Virgin Islands`, as a country.